core/sql/ustat/hs_yacc.y

```diff
@@ -60,6 +60,8 @@ extern THREAD_P HSGlobalsClass *hs_globals_y;
 extern int yylex(YYSTYPE * lvalp, void* scanner);
 //extern int yylex();
 
+extern NABoolean isUpdatestatsStmt;
```

Comment:
Good point. I'll move it to hs_globals.

On Mon, Sep 21, 2015 at 11:11 AM, DaveBirdsall <notifications@github.com>
wrote:

> In core/sql/ustat/hs_yacc.y
> <https://github.com/apache/incubator-trafodion/pull/87#discussion_r39990031>
> :
>
> > @@ -60,6 +60,8 @@ extern THREAD_P HSGlobalsClass *hs_globals_y;
> >  extern int yylex(YYSTYPE * lvalp, void* scanner);
> >  //extern int yylex();
> >
> > +extern NABoolean isUpdatestatsStmt;
>
> There's no object we can stick this global inside of?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/87/files#r39990031>.
>



-- 
Regards, --Qifan